src/python/grpcio_test/setup.py

```diff
@@ -48,8 +48,13 @@
 
 _PACKAGE_DATA = {
     'grpc_interop': [
```

Comment:
From offline discussion: this was changed to a tuple earlier, but caused type assertion errors in setuptools. It remains a list thus.